Code/Framework/AzToolsFramework/Tests/FocusMode/EditorFocusModeSelectionTests.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Tests/FocusMode/EditorFocusModeFixture.h>
+
+#include <AzCore/Component/TransformBus.h>
+#include <AzCore/std/string/string.h>
+
+#include <AzFramework/Viewport/ViewportScreen.h>
+
+#include <AzManipulatorTestFramework/AzManipulatorTestFramework.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzManipulatorTestFramework/DirectManipulatorViewportInteraction.h>
+#include <AzManipulatorTestFramework/ImmediateModeActionDispatcher.h>
+#include <AzManipulatorTestFramework/IndirectManipulatorViewportInteraction.h>
+
+#include <AzToolsFramework/Component/EditorComponentAPIBus.h>
+#include <AzToolsFramework/Manipulators/LinearManipulator.h>
+#include <AzToolsFramework/Manipulators/ManipulatorManager.h>
+#include <AzToolsFramework/ViewportSelection/EditorVisibleEntityDataCache.h>
+
+
+namespace AzToolsFramework
+{
+    using EditorFocusModeSelectionFixture = UnitTest::IndirectCallManipulatorViewportInteractionFixtureMixin<EditorFocusModeFixture>;
+
+    TEST_F(EditorFocusModeSelectionFixture, EditorFocusModeSelectionTests_SelectEntity)
+    {
+        auto selectedEntitiesBefore = GetSelectedEntities();
+        EXPECT_TRUE(selectedEntitiesBefore.empty());
+
+        // calculate the position in screen space of the initial entity position
+        const auto carScreenPosition = AzFramework::WorldToScreen(CarEntityPosition, m_cameraState);
```

Comment:
question: Have you checked where the position of the camera is if this isn't working? I usually do a quick sanity check in the editor viewport first to get a rough idea. There are lots of helpers then in CameraState.h/cpp to set the camera position/transform accordingly

Reply:
I had, but I believe the issue was with having multiple bounds components around. I managed to make it work now.